coldfront/core/project/views.py

```diff
@@ -4162,13 +4162,15 @@ def form_valid(self, form):
                 allocation_obj = Allocation.objects.get(project=project_obj)
```

Comment:
I'd move the `try`-`except` into the later `if status == 'Complete'` block, after the `ProjectUser`'s status gets updated.

So all the updates to the request happen in the same place, and we only update the allocation-related objects if the `ProjectUser` successfully got updated.